packages/admin/src/targetGroups/addContacts/AddContactsGridSelect.gql.ts

```diff
@@ -12,8 +12,14 @@ const targetGroupBrevoContactsFragment = gql`
 `;
 
 export const manuallyAssignedBrevoContactsGridQuery = gql`
-    query ManuallyAssignedBrevoContactsGrid($offset: Int, $limit: Int, $email: String, $targetGroupId: ID!) {
-        manuallyAssignedBrevoContacts(offset: $offset, limit: $limit, email: $email, targetGroupId: $targetGroupId) {
+    query ManuallyAssignedBrevoContactsGrid($offset: Int, $limit: Int, $email: String, $targetGroupId: ID!, $onlyManuallyAssigned: Boolean) {
+        assignedBrevoContacts(
+            offset: $offset
+            limit: $limit
+            email: $email
+            targetGroupId: $targetGroupId
+            onlyManuallyAssigned: $onlyManuallyAssigned
```

Comment:
You could set this to `true` here directly since this query should only be used for manuasllyAssignedContacts as the naming suggests

(same for the all query)